WordPress/src/main/res/layout/people_invite_fragment.xml

```diff
@@ -84,26 +83,49 @@
             android:background="?android:selectableItemBackground"
             android:orientation="vertical">
 
-            <org.wordpress.android.widgets.WPTextView
-                android:layout_width="wrap_content"
+            <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:ellipsize="end"
-                android:text="@string/role"
-                android:textColor="@color/grey_dark"
-                android:textSize="@dimen/text_sz_large"
-                tools:text="@string/role"/>
+                android:orientation="horizontal">
+
+                <org.wordpress.android.widgets.WPTextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:ellipsize="end"
+                    android:gravity="center_vertical"
+                    android:text="@string/role"
+                    android:textColor="@color/grey_dark"
+                    android:textSize="@dimen/text_sz_large"
+                    tools:text="@string/role" />
+
+                <ImageView
+                    android:id="@+id/imgInfo"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:paddingLeft="@dimen/margin_small"
+                    android:paddingRight="@dimen/margin_small"
+                    android:src="@drawable/dashicon_info" />
+            </LinearLayout>
 
             <org.wordpress.android.widgets.WPTextView
                 android:id="@+id/role"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="5dp"
                 android:layout_marginLeft="1dp"
+                android:layout_marginTop="5dp"
+                android:background="@drawable/invites_border"
```

Comment:
Probably on the subjective side of things but, I'd suggest to not make the "dropdown" have a border like this one since the styling is kinda foreign both to android and the rest of the app. Instead, I would suggest removing the border and having the arrow right after the TextView. Thanks!